scylla-cdc/src/checkpoints.rs

```diff
@@ -209,7 +210,7 @@ impl CDCCheckpointSaver for TableBackedCheckpointSaver {
     ) -> anyhow::Result<Option<chrono::Duration>> {
         Ok(self
             .session
-            .query(
+            .query_unpaged(
                 format!(
                     "SELECT time FROM {}
                     WHERE stream_id = ?
```

Comment:
:thinking: ditto about SELECT with `query_unpaged`.

Reply:
Again, this is fine.